SQL: document that the Query action requires the SQL integration (#46738)
home-assistant/home-assistant.github.io: source/_actions/sql.query.markdown

The **Query** action runs a read-only `SELECT` query against a database and returns the rows it finds.

This is useful when you want an automation or script to pull data on demand, for example to look up recent history or summarize values, without adding a separate sensor for each query. Only `SELECT` statements are allowed.

> {% important %}
> The **Query** action only becomes available after the [SQL integration](/integrations/sql/) is set up. Setting up the integration adds at least one SQL sensor, and that is what makes this action appear. If the **SQL: Query** action is missing, add the SQL integration first from {% my integrations title="**Settings** > **Devices & services**" %}, or configure it in YAML.
> {% endimportant %}

Reviewer: Copilot has a point. Would be good to refer that an SQL sensor must be added:
```suggestion
{% important %}>
The **Query** action is only available after you set up the [SQL integration](/integrations/sql/) by adding at least one SQL sensor.
{% endimportant %}
```
